pkg/scalers/tls_config.go

```diff
@@ -0,0 +1,38 @@
+package scalers
```

Comment:
Could you please move this out of the `scalers` package? `pkg/util` or `pkg/util/tls` etc

Reply:
This is done. Since this is generic, moving this to util will help to manage other scalers better. Thanks!